code/modules/events/supermatter_surge.dm

```diff
@@ -80,8 +80,8 @@
 
 	end_when = rand(SURGE_DURATION_MIN, SURGE_DURATION_MAX)
 
-/datum/round_event/supermatter_surge/announce()
-	priority_announce("The Crystal Integrity Monitoring System has detected unusual atmospheric properties in the supermatter chamber and energy output from the supermatter crystal has increased significantly. Engineering intervention is required to stabilize the engine.", "Class [surge_class] Supermatter Surge Alert", 'sound/machines/engine_alert3.ogg')
+/datum/round_event/supermatter_surge/announce(fake)
+	priority_announce("The Crystal Integrity Monitoring System has detected unusual atmospheric properties in the supermatter chamber, energy output from the supermatter crystal has increased significantly. Engineering intervention is required to stabilize the engine.", "Class [surge_class] Supermatter Surge Alert", 'sound/machines/engine_alert3.ogg')
```

Comment:
Why replace the and with a comma, isn't that less correct?

Reply:
they're both just about as correct as each other

Reply:
It was originally CIMS as an acronym, but now that the full name is used the sentence was feeling a little long without a pause.

Reply:
a comma is not a direct substitute for "and" but I suppose its a vanishingly minor quibble

Reply:
this sentence is completely grammatically correct with either "and" or a comma.